e2e/namespace/upgrade/cli_upgrade_test.go

```diff
@@ -51,7 +51,7 @@ func TestOperatorUpgrade(t *testing.T) {
 		Expect(os.Setenv("KAMEL_BIN", kamel)).To(Succeed())
 
 		// Should both install the CRDs and kamel in the given namespace
-		Expect(Kamel("install", "-n", ns, "--olm=false", "--force").Execute()).To(Succeed())
+		Expect(Kamel("install", "-n", ns, "--olm=false", "--force", "--base-image", defaults.BaseImage()).Execute()).To(Succeed())
```

Comment:
I think this is not required. As default, it already use that base image.

Reply:
I may have missed something but my understanding is that it is needed because the installation installs the previous version of the operator which uses adoptopnjdk as base image, then it performs an upgrade.

Reply:
Ah, sorry, did not notice it was related to previous version. It's okey then, altough we may need to revert the change in future versions.